packages/menu-bar/test/sub-menu.common.js

```diff
@@ -607,6 +607,20 @@ describe('accessibility', () => {
     });
   });
 
+  it('should set role attribute on host element in tabNavigation', async () => {
```

Comment:
Moved the "accessibility" test suite to `a11y.common.js` in #7530, please move these tests there.

Reply:
Done